### DISCUSSION TITLE
Configure block size

### DISCUSSION THREAD

**Discussion by ramazanjusip** [minio/minio#12767] [Q&A] [answered]
H!

How can I configure block size of data being written to HDD disk?
Does MinIO use HDD disk's preformatted block size?
How can I check which block size MinIO put objects?

**Comment by harshavardhana** [comment, marked as answer]
You cannot why do you want to do that?

**Reply by ramazanjusip** [reply]
We're going to write 3-8MB chuncks of video to minio cluster with HDD drives and speed up IO increasing block size.
